pkg/workloads/spark_job/spark_util.py

```diff
@@ -530,9 +525,13 @@ def validate_transformer(column_name, test_df, ctx, spark):
                         + inferred_python_type,
                     )
 
+            ctx.write_metadata(
+                    transformed_column["id"], transformed_column["metadata_key"], {"type": inferred_python_type}
+                )
+
         try:
```

Comment:
I feel like maybe the whole `if hasattr(trans_impl, "transform_python"):` section should be within the `try`? @vishalbollu what do you think?

Reply:
yes I agree, then we can add more context to the potential exception in transform_python